packages/dbml-core/src/model_structure/endpoint.js

```diff
@@ -9,7 +9,8 @@ class Endpoint extends Element {
 
     this.schemaName = schemaName;
     this.tableName = tableName;
-    this.fieldName = fieldName;
+    this.fieldName = fieldName; // can be an array of names
```

Comment:
As I see at all the places in this PR, this is always an array of names. 

Suggest changing `fieldName` to `fieldNames` instead.